### PR TITLE
fixed blank Home Screen

### PR DESCRIPTION
**Pull request by B6Amine**
I manage to fix the home screen when it appears blank for the users who doesn't have pets or reservations . Here's now a text that appears . 
empty screen

### CODE REVIEW

**Review by karinerafla — CHANGES_REQUESTED** [review]
Please make sure the CI passes before I can review.

File: app/src/main/java/com/android/PetPamper/ui/screen/users/CaptureImage.kt
Comment (karinerafla) [inline app/src/main/java/com/android/PetPamper/ui/screen/users/CaptureImage.kt:1]: Appropriately checks camera permissions.

Implementation for creating image files and managing URIs using FileProvider is correct and ensures secure access, maybe a bit of error handling here would've been useful. 

The process of creating the image file and URI is duplicated in the permission check and permission request sections. could be refactored into a single method to reduce duplication.

**Review by karinerafla — APPROVED** [review]
Unfortunately since we are short on time and the code functionality is as needed, will approve the branch without it having passed the CIs (does not affect the app). 

This is a good implementation of the usage and camera permissions. However I do not see how this PR fixes the home screen issue... The only modified git is showing me is the CaptureImage.kt file.

**Comment by B6Amine** [comment]
> Unfortunately since we are short on time and the code functionality is as needed, will approve the branch without it having passed the CIs (does not affect the app).
> 
> This is a good implementation of the usage and camera permissions. However I do not see how this PR fixes the home screen issue... The only modified git is showing me is the CaptureImage.kt file.

it fixes the home screen issue where it does not appear blank when u do not have a pets or reservation u have a text message who says empty as u can see in the screenshot

**Pull request closed without merging**